<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KexiFindDialog</class>
 <widget class="QDialog" name="KexiFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel2_2">
     <property name="text">
      <string comment="Look in field:">&amp;Look in:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_lookIn</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel2_2_2">
     <property name="text">
      <string>&amp;Search:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_search</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="m_wholeWords">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string comment="Match: Whole words only">&amp;Whole words only</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Fi&amp;nd:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_textToFind</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="m_lookIn">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KHistoryComboBox" name="m_textToFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_replaceLbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string comment="Replace all occurences with">Re&amp;place with:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_textToReplace</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="m_promptOnReplace">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string>Prompt on replace</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="KHistoryComboBox" name="m_textToReplace">
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel2_2_3">
     <property name="text">
      <string>&amp;Match:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_match</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="m_caseSensitive">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="text">
      <string comment="Match: Case sensitive">C&amp;ase sensitive</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KComboBox" name="m_match">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <item>
      <property name="text">
       <string comment="Match: Any Part of Field">Any Part of Field</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Match: Whole Field">Whole Field</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Match: Start of Field">Start of Field</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KComboBox" name="m_search">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <item>
      <property name="text">
       <string comment="Search Up">Up</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Search Down">Down</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="Look In All Rows">All Rows</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" rowspan="8">
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="KPushButton" name="m_btnFind">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_btnReplace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_btnReplaceAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="Replace All Occurences">Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="m_messageLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_textToFind</tabstop>
  <tabstop>m_textToReplace</tabstop>
  <tabstop>m_btnFind</tabstop>
  <tabstop>m_btnClose</tabstop>
  <tabstop>m_btnReplace</tabstop>
  <tabstop>m_btnReplaceAll</tabstop>
  <tabstop>m_lookIn</tabstop>
  <tabstop>m_search</tabstop>
  <tabstop>m_match</tabstop>
  <tabstop>m_caseSensitive</tabstop>
  <tabstop>m_wholeWords</tabstop>
  <tabstop>m_promptOnReplace</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnClose</sender>
   <signal>clicked()</signal>
   <receiver>KexiFindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>